R/filter_nth_entry.R

```diff
@@ -0,0 +1,50 @@
+#' Filter dataframe for nth entry
+#'
+#' Filter a dataframe for the nth entry of each subject in it.
+#' A typical use cases would be to filter a dataset for the first or last measurement of a subject.#'
+#'
+#' @param data the data.frame to filter
+#' @param ID_column character column identifying subjects
+#' @param entry_column character column identifying order of entries.
+#' That column can by of types Date, numeric, or any other type suitable for order()
+#' @param n integer number of entry to keep after ordering
+#' @param reverse_order logical when TRUE sorts entries last to first before filtering
+#'
+#' @return data.frame with <= 1 entry per subject
+#'
+#' @examples
+#' data <- data.frame(list(ID = rep(1:5, 3), encounter = rep(1:3, each=5), value = rep(4:6, each=5)))
+#'
```

Comment:
Seems like the example is missing. Please add.

Reply:
Updated code and mnual to include examples